packages/react-data-grid/src/common/types.ts

```diff
@@ -166,7 +166,7 @@ export interface RowsContainerProps {
   children: React.ReactElement;
 }
 
-export interface RowRendererProps<TRow> {
+export interface IRowRendererProps<TRow> {
```

Comment:
`IRowRendererProps<TRow>` vs `IRowProps<TRow>`, wdyt?

Then for this line:
https://github.com/adazzle/react-data-grid/pull/1787/files#diff-c77bf2dcbfe5c1705d249cd2d09835aeR8
```typescript
export default class Row<R> extends React.Component<RowProps<R>> implements IRowRenderer
```

Reply:
Mmh, I'm not sure, it's shared across multiple components I think, not just the `Row` component.

Reply:
Then I guess we should leave this naming for now and revisit later.